code/API_definitions/number-recycling.yaml

```
@@ -73,7 +73,7 @@ externalDocs:
   description: Product documentation at Camara
   url: https://github.com/camaraproject/NumberRecycling
 servers:
-  - url: '{apiRoot}/number-recycling/vwip'
+  - url: '{apiRoot}/number-recycling/0.1.0-alpha.1'
```

See rules for API versioning [here](https://github.com/camaraproject/Commonalities/blob/main/documentation/API-design-guidelines.md#53-api-versions-throughout-the-release-process)
```suggestion
  - url: '{apiRoot}/number-recycling/0.1alpha1'
```

---

I think `0.1alpha1` can be `v0.1alpha1`